### PR TITLE
add technique-of relation type

### PR DESCRIPTION
**Pull request by ChanochShayner**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
related: https://github.com/demisto/etc/issues/36138

## Description
add `technique-of` relation type.

## Screenshots
image

### CODE REVIEW

**Review by David-BMS — APPROVED** [review]
@ChanochShayner Make sure to reference to an issue.

File: Packs/Base/Scripts/CommonServerPython/CommonServerPython.py [hunk 1]
```diff
@@ -4604,6 +4604,7 @@ class Relations(object):
         SENT_TO = 'sent-to'
         SIMILAR_TO = 'similar-to'
         SUB_DOMAIN_OF = 'sub-domain-of'
+        SUB_TECHNIQUE_OF = 'subtechnique-of'
```
Comment (orhovy) [inline Packs/Base/Scripts/CommonServerPython/CommonServerPython.py:4607]:
```suggestion
        SUB_TECHNIQUE_OF = 'subtechnique-of'
        PARENT_TECHNIQUE_OF = 'parent-technique-of'
```

File: Packs/Base/ReleaseNotes/1_10_4.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
+#### Scripts
+##### CommonServerPython
+- Added support for relations between indicators.
```
Comment (orhovy) [inline Packs/Base/ReleaseNotes/1_10_4.md:4]: update to accurate rn

Comment (ChanochShayner) [reply Packs/Base/ReleaseNotes/1_10_4.md]: done

**Review by orhovy — APPROVED** [review]

**Pull request merged**